kymatio/scattering2d/backend/__init__.py

```diff
@@ -61,9 +61,9 @@
 
 
 if BACKEND == 'torch':
-    from .backend_torch import *
+    from .torch_b import *
```

Comment:
why this naming change?

Reply:
b for backend. I'd like that the backends related to a frontend (they will be hard to disentangle)  start with a similar prefix, what do you think?

Reply:
I've chosen the last naming convention (i.e., adding *_backend* at the end)

Reply:
:ok_hand: